### DISCUSSION TITLE
adding angles to line

### DISCUSSION THREAD

**Discussion by armeya25** [highfestiva/finplot#432] [Q&A] [answered]
![screenshot](https://github.com/highfestiva/finplot/assets/135110431/5e9d1326-b648-42bd-993e-a8d358e5cc3a)
how do i add angle to horizontal line at certain location/bar
see attachment plz
thanks in advance.

**Comment by armeya25** [comment]
its purple line on which i want to draw a line at certain angle say 30 degrees
line is to start from high of that candle to some point up/down direction.

how do i do it

**Comment by highfestiva** [comment]
There is an `finplot.add_line()` function, see the `line.py` example.

**Reply by armeya25** [reply]
i wish to add angle on that line,
working on GAAN angles for now
for example,
on higher high point i wish to draw angle of 30

**Reply by highfestiva** [reply, marked as answer]
finplot does not have any fixed angle feature, since it makes little sense to disregard scale and price in a price chart. If you want it, you'd have to code that yourself.
